Virtusize/Tests/VirtusizeAPIServiceTests.swift

```diff
@@ -618,17 +617,16 @@ class VirtusizeAPIServiceTests: XCTestCase {
 
     func testGetUserBodyRecommendedSize() {
         let expectation = self.expectation(description: "Virtusize.getUserBodyRecommendedSize reaches the callback")
-        var actualRecommendedSize: BodyProfileRecommendedSize?
+        var actualRecommendedSizes: [BodyProfileRecommendedSize]?
 
-		VirtusizeAPIService.session = MockURLSession(
-            data: "{\"sizeName\": \"35\"}".data(using: .utf8),
+        VirtusizeAPIService.session = MockURLSession(
+            data: "[{\"sizeName\": \"35\"}]".data(using: .utf8),
             urlResponse: nil,
             error: nil
         )
 
         DispatchQueue.global().async {
-
-            actualRecommendedSize = VirtusizeAPIService.getBodyProfileRecommendedSizeAsync(
+            actualRecommendedSizes = VirtusizeAPIService.getBodyProfileRecommendedSizeAsync(
```

Comment:
The only fix needed to make was caused by the return value of getBodyProfileRecommendedSizeAsync becoming an array. There was no reason to comment out the other unit tests.